tests/test_github_tags.py

```diff
@@ -0,0 +1,335 @@
+# fetchcode is a free software tool from nexB Inc. and others.
+# Visit https://github.com/nexB/fetchcode for support and download.
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and http://aboutcode.org
+#
+# This software is licensed under the Apache License version 2.0.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: 
+# http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+from unittest import mock
+
+from fetchcode import github_tags
+
+@mock.patch('fetchcode.github_tags.requests.get')
+def test_github_tags_for_not_empty_list(mock_get):
+    url = 'https://github.com/nexb/scancode-toolkit'
+    mock_get.return_value.json.return_value =[
```

Comment:
Can you move this big thing to a separate json file?

Reply:
Cool, will do so

Reply:
Done @majurg :D